rbac/management/group/view.py

```diff
@@ -464,15 +472,22 @@ def add_service_accounts(
             if len(invalid_service_accounts) > 0:
                 raise ServiceAccountNotFoundError(f"Service account(s) {invalid_service_accounts} not found.")
 
+    def add_service_accounts(
+        self,
+        group: Group,
+        service_accounts: Iterable[dict],
+        org_id: str = "",
+    ) -> Group:
+        """Add service accounts to the group."""
```

Comment:
Looks like the type signature is wrong here 

```suggestion
    ) -> Tuple[Group, List[Principal]]:
```

Reply:
Thanks, done 👍